srcpkgs/x13s-firmware/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'x13s-firmware'
+pkgname=x13s-firmware
+version=20240312
+revision=1
+_commit=d1e35f1c10398994f4a675c9c42f089b88bc806d
+archs="aarch64*"
+build_style=fetch
+short_desc="Firmware for ThinkPad X13s"
+maintainer="classabbyamp <void@placeviolette.net>"
+license="custom:idk"
```

Comment:
this may need to be a restricted package, as this comes from arch linux arm, who I think have extracted it from windows/lenovo. licencing seems dubious at best.

Reply:
My brief searches on a690_gmu.bin show that up until recently a660_gmu.bin was the recommended firmware for the x13s. Are there any details on where the x13s-alarm project pulled in a690_gmu.bin from / why they switched to it?

Reply:
my search showed that a660 works but a690 is the exact firmware for it

Reply:
decided to drop it and extract the firmware from lenovo directly, but still unsure on the licence, as there's no indication on [lenovo's site](https://pcsupport.lenovo.com/us/en/products/laptops-and-netbooks/thinkpad-x-series-laptops/thinkpad-x13s-type-21bx-21by/downloads/driver-list/)